HyperPlay-Gaming/hyperplay-desktop-client
# [Feat] Add rpcRaw endpoint to proxy server

Add rpcRaw endpoint to proxy server


### Testing
1. Clone https://github.com/HyperPlay-Gaming/SiweTest
2. Start this branch
4. run `yarn` and then  `yarn test` in the root directory of SiweTest
5. Change max batch count to 1 in SiweTest/test.ts
6. run `yarn test` again
7. Sign message with hyperplay connected wallet
8. See that signature was verified correctly for siwe test and no errors on hyperplay side
9. Repeat with metamask mobile, wallet connect, metamask extension

**Make sure you're on the correct proxy server commit**
---

Use the following Checklist if you have changed something on the Backend or Frontend:
